Gh-361: Improve Gaffer Gremlin examples (#362)
gchq/gaffer-docker: docker/gaffer-gremlin/README.md

A demo/example using the tinkerpop 'modern' dataset and accumulo backed Gaffer
is available under the `example` directory. This can be ran using docker compose

> to deploy the containers then, the provided jupiter notebooks demonstrate how to

Reviewer: ```suggestion
to deploy the containers. The provided jupyter notebooks demonstrate how to
```

> connect and some basic queries on the data using `gremlinpython` or via

Reviewer: ```suggestion
connect to your graph and some basic Gremlin queries on the data using `gremlinpython` or via
```
I think this sentence just needs a little clarity. This is just one way of doing that I think.

`graph-notebook` (note this requires the [graph-notebook extension](https://github.com/aws/graph-notebook)).

To run the example first source the environment file for the image versions

> e.g. `accumulo2.env` then use docker compose to start up the containers:

Reviewer: ```suggestion
e.g. `accumulo2.env`, then use docker compose to start up the containers:
```

```bash
source ../accumulo2.env

> docker compose up -d

Author: This should have been addressed now its moved to a proper start script to deploy and load the data
